src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,68 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String DIFFERENCE = "result";
+    private static final int RESULT_ARRAY_SIZE = 3;
+    private static final int INDEX_SUPPLY = 0;
+    private static final int INDEX_BUY = 1;
+    private static final int INDEX_RESULT = 2;
+    private static final int INDEX_NAMES = 0;
+    private static final int INDEX_PRICE = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File inputFile = new File(fromFileName);
+        File outputFile = new File(toFileName);
+        String[] resultData = getCountResult(getStringFile(inputFile));
+        writeToFile(resultData, outputFile);
+    }
+
+    private List<String> getStringFile(File file) {
+        try {
+            return (Files.readAllLines(file.toPath()));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file " + e);
+        }
+    }
+
+    private void writeToFile(String[] data, File fileName) {
+        if (fileName.exists()) {
+            fileName.delete();
+        }
+        for (String str : data) {
+            try {
+                fileName.createNewFile();
```

Comment:
```suggestion
                Files.write(fileName.toPath(), "");
```
What about this option?